relayer/go.mod

```diff
@@ -196,3 +139,13 @@ require (
 )
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+
+replace github.com/smartcontractkit/chainlink-starknet/relayer => ../../chainlink-starknet/relayer
+
+replace github.com/smartcontractkit/chainlink-env => ../../chainlink-env
+
+replace github.com/smartcontractkit/chainlink-testing-framework => ../../chainlink-testing-framework
+
+replace github.com/smartcontractkit/chainlink => ../../chainlink
+
+replace github.com/smartcontractkit/chainlink/integration-tests => ../../chainlink/integration-tests
```

Comment:
We should have an pre-prepared  but commented out `go.work` file added to the repo, so we can do those replaces there and avoid doing them in module `go.mod` so we don't accidentally forget it.